### PR TITLE
feat: create list page

### PR DESCRIPTION
**Pull request by yoslie**
## Overview
create list page

### Preview

https://github.com/jabardigitalservice/j-site-builder/assets/65396086/0147b36e-662f-4ed6-8eba-d32d09c6c38c

### Related
[S5A2 - Membuat List Page](https://app.clickup.com/t/9003239225/CES-1304)

## Evidence
title: create list page
project: J-Site
participants: @yoslie @doohanas @Ibwedagama

### CODE REVIEW

**Review by doohanas — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true,
     "editor.codeActionsOnSave": {
-      "source.fixAll.eslint": true,
-      "source.fixAll.stylelint": true
+      "source.fixAll.eslint": "explicit",
+      "source.fixAll.stylelint": "explicit"
```
Comment (Ibwedagama) [inline .vscode/settings.json:16]: hello @yoslie can you explain what these changes does to the linter?

File: components/Page/List.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
+<template>
+  <li
+    v-for="(item, index) in props.data"
+    :key="index"
```
Comment (Ibwedagama) [inline components/Page/List.vue:4]: hello @yoslie please use item id or item.title for key @yoslie 👍🏼

**Review by Ibwedagama — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (yoslie) [reply .vscode/settings.json:16]: this changes keep showing automatically lately, should I just keep ignore this? @Ibwedagama

File: components/Page/List.vue (hunk 2 above)
Comment (yoslie) [reply components/Page/List.vue]: sure thanks 👍

**Pull request merged**